envoy/filesystem/filesystem.h

```diff
@@ -169,6 +169,9 @@ struct DirectoryEntry {
   // target. For example, if name_ is a symlink to a directory, its file type will be Directory.
   FileType type_;
 
+  // The file size in bytes.
```

Comment:
nit: I don't feel a compelling need to document this variable any more than it's well-chosen name :) 

No worries if you prefer to leave it though.

Reply:
I'd rather leave it because I don't know what to do with the spacing otherwise, haha. A blank line between variables is weird, but with no blank line the comment above the *other* variable seems to cover both.